app/jobs/fastly_log_processor.rb

```diff
@@ -0,0 +1,75 @@
+require 'zlib'
+FastlyLogProcessor = Struct.new(:bucket, :key) do
+  def perform
+    counts = download_counts
+
+    unless Gemcutter::ENABLE_FASTLY_LOG_PROCESSOR
```

Comment:
@qrush We still want to enqueue the job & process the log regardless.

This feature flag is toggling whether we actually update the counts in redis; or just log the counts for debugging.

NB this is a temporary feature flag while we dark-launch log processing. I'll remove it once we're confident it's working well.